WordPress/WordPressTest/RemoteFeatureFlagTests.swift

```diff
@@ -0,0 +1,79 @@
+import XCTest
+@testable import WordPress
+
+class RemoteFeatureFlagTests: XCTestCase {
+
+    override func setUpWithError() throws {
+        UserDefaults.standard.removeObject(forKey: RemoteFeatureFlagStore.Constants.CachedFlagsKey)
+        UserDefaults.standard.removeObject(forKey: RemoteFeatureFlagStore.Constants.DeviceIdKey)
+    }
+
+    func testThatDeviceIdIsTheSameForEveryInstanceOfTheStore() {
+        let mock = MockFeatureFlagRemote()
+        var deviceId = ""
+        mock.deviceIdCallback = {
```

Comment:
Nitpicky: I'd consider adding an expectation so that we know the `deviceIDCallback` is called twice.  The reason is that if the callback is never executed this test will never fail.

Something like:

```swift
func testThatDeviceIdIsTheSameForEveryInstanceOfTheStore() {
    let callbackExecuted = expectation(description: "Callback executed twice")
    callbackExecuted. expectedFulfillmentCount = 2

    let mock = MockFeatureFlagRemote()
    var deviceId = ""
    mock.deviceIdCallback = {
        deviceId == "" ? deviceId = $0 : XCTAssertEqual(deviceId, $0)
    }

    RemoteFeatureFlagStore(remote: mock).update()
    RemoteFeatureFlagStore(remote: mock).update()

    waitForExpectations(timeout: 0.1)
}
```

Reply:
This seems like a good extra check, so added in 7ba7d0946ec64e0e86576eb0cfc92854203cc07d :)